include/aws/auth/credentials.h

```diff
@@ -406,6 +406,23 @@ struct aws_credentials_provider_chain_default_options {
     struct aws_client_bootstrap *bootstrap;
 };
 
+/**
+ * For language binding to provide credentials from that language
```

Comment:
Given the generality here, there's no need to talk about language bindings.

Reply:
Yeah, changed words there